core/src/main/scala/cats/data/Cokleisli.scala

```diff
@@ -14,12 +14,13 @@ final case class Cokleisli[F[_], A, B](run: F[A] => B) { self =>
   /**
     * Example:
     * {{{
-    * scala> import cats.Id, cats.implicits._
-    * scala> val x : Id[Int] = 42
-    * scala> def before(x: Int) = x + 1
-    * scala> def after(x: Int) = x - 1
-    * scala> val example : Cokleisli[Id,Int,Int] = Cokleisli((f: Id[Int]) => f.extract)
-    * scala> example.dimap(before)(after) == 42
+    * scala> import cats._, data._
+    * scala> val f = Cokleisli((xs: NonEmptyList[Int]) => xs.reverse.head)
+    * f: cats.data.Cokleisli[cats.data.NonEmptyList,Int,Int] = Cokleisli(<function1>)
```

Comment:
I'd omit these toString renderings, just like you omitted the output of the defs.  The other examples in Cats tend to omit the trivial outputs and focus on the setup and the res0.

Reply:
@rossabaker gotcha :)